runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2910,6 +2910,16 @@ gcInitializeDefaults(J9JavaVM* vm)
 
 	initializeVerboseFunctionTableWithDummies(&extensions->verboseFunctionTable);
 
+#if defined(J9VM_GC_MODRON_SCAVENGER)
+	/* enable enableScanConstantPoolObjectInScavenge if Java 11 or above */
+	if (J2SE_VERSION(vm) < J2SE_V11) {
+		extensions->enableScanConstantPoolObjectInScavenge = false;
+	} else {
+		extensions->enableScanConstantPoolObjectInScavenge = true;
+	}
```

Comment:
We'll need this for Java 8 as well.  The goal is to enable this so we can move to the openjdk methodhandle implementation which won't have the tenured MT & MHs